drivers/spi/spi_gd32.c

```diff
@@ -195,6 +253,115 @@ static int spi_gd32_frame_exchange(const struct device *dev)
 	return spi_gd32_get_err(cfg);
 }
 
+#ifdef CONFIG_SPI_GD32_DMA
+static void spi_gd32_dma_callback(const struct device *dmadev, void *arg,
```

Comment:
Suggestion: change `dmadev` to `dma_dev`,

Reply:
Applied your suggestion, Thank you.